solutions/USACO open18-team-tic-tac-toe.mdx

```diff
@@ -2,7 +2,7 @@
 id: usaco-831
 source: USACO Bronze 2018 Open
 title: Team Tic Tac Toe
-author: Jesse Choe, Benjamin Qi
+author: Jesse Choe, Kevin Sheng, Benjamin Qi
```

Comment:
🤦

Reply:
so should i? i mean i did so something lol